src/style.css

```
@@ -0,0 +1,80 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+@media (min-width: 900px) {
+  .container > .block {
+    flex-basis: 33.33%;
+  }
```

Now let's make the next step and not use selectors with increased specificity. Just class is enough here.
```suggestion
  .block {
    flex-basis: 33.33%;
  }
```

---

ok)

---

https://prnt.sc/1rp4c97
without .container >